2-run-kobiton-test.md

```diff
@@ -0,0 +1,123 @@
+# Run Kobiton Automation Test on Travis CI
+This guide will demonstrate How to run your Kobiton test on Travis CI and how to secure your secret environment variables
+
+## Prerequisites
+  - Github account.
+  - Kobiton installed.
+  - Kobiton account.
+
+## 1. Prepare Kobiton configuration for executing automation testing
+  - Go to https://portal.kobiton.com
+
+### 1.1 Get Kobiton Username
+  - In the upper right hand corner, click on User icon and in the drop down menu, click on `Profile`.  
+![](assets/2_kobiton_profile.jpg)
+
+  - You should see your username.  
+![](assets/2_kobiton_username.jpg)
+
+### 1.2 Get Kobiton API key
+  - Click on User icon and select `Settings` on the drop down menu.
+![](assets/2_kobiton_profile_2.jpg)  
+
+  - Copy your API key under `API Keys`.  
+![](assets/2_kobiton_apikey.jpg)
+
+## 2. Configure environment variables in your .travis.yml file
+There are several methods to attach an enviroment variable to Travis CI.
+In this guide, we will add our Username and API key to `.travis.yml` file. The API Key will be encrypted for security purposes.
```

Comment:
"The API Key will be encrypted for security purposes." Duplicate with "Encrypt API key for security purposes:" ?